src/models/Observation.js

```diff
@@ -108,19 +147,34 @@ class Observation {
     primaryKey: "uuid",
     properties: {
       uuid: "string",
+      captive_flag: "bool?",
       comments: "Comment[]",
       created_at: { type: "string?", mapTo: "createdAt" },
       description: "string?",
+      geoprivacy: "string?",
+      id: "int?",
       identifications: "Identification[]",
       latitude: "double?",
       longitude: "double?",
       observationPhotos: "ObservationPhoto[]",
-      observationSounds: "ObservationSound[]",
+      observationSound: "ObservationSound?",
```

Comment:
Oh, interesting. I think the UX confused me here -- I remember Abhas saying we only wanted to let users upload one sound at a time. And since we don't have anywhere to display existing sounds in the current UX, I didn't think there was a need for multiple sounds. I'll change this back.